apps/src/templates/sectionsRefresh/SectionsSetUpContainer.jsx

```diff
@@ -306,7 +309,15 @@ export default function SectionsSetUpContainer({
           {tooltip}
         </div>
       ),
-      () => <div>{i18n.coteacherAddInfo()}</div>,
+      () => (
+        <CoteacherSettings
+          sectionInstructors={sections[0].sectionInstructors}
```

Comment:
At first glance this looks suspiciously like it might fail when editing something other than one's first section, but then I see this pattern repeated elsewhere in this file. Is `sections` always a one-element array? If so, any reason we couldn't get rid of the array and switch all the `sections[0]` references to simply `section`?

Reply:
https://codedotorg.slack.com/archives/C045UAX4WKH/p1698094245815849

Reply:
This is because we eventually want to allow multiple sections to be created on the same form. Let's keep this around